manifests/input.yaml

```diff
@@ -652,29 +652,29 @@ work/input/make_ryf/make_ryf.py:
     binhash: 9fbb6a9f6bec50ccb2afafecae9a5ff7
     md5: c5ef2aeff9516ec76c80e76aef08d352
 work/input/ocean_hgrid.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/ocean_hgrid.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_hgrid.nc
   hashes:
-    binhash: b4895a8dfb1d02da1c5c8b114bbcff05
+    binhash: 7f9cf6930cda20f2ba304eef45c2a97d
     md5: 51f58be0f4ea6da2cb438a893f95c689
 work/input/ocean_temp_salt.res.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/ocean_temp_salt.res.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_temp_salt.res.nc
   hashes:
-    binhash: dba1ab70ca11f3758336dd642a027489
+    binhash: 4812809ecd2abcc47351263cd2c66821
     md5: c5f7e60b5427a4442f111adc65a2d067
 work/input/ocean_vgrid.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/ocean_vgrid.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_vgrid.nc
   hashes:
-    binhash: 4d33e34fa792d43dc7eba7d2f22a05d1
+    binhash: 30af5058d31d9133770757144f6bd7cf
     md5: a1e0b0b0adc363506ff17a362e83f64a
 work/input/salt_sfc_restore.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/salt_sfc_restore.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/salt_sfc_restore.nc
   hashes:
-    binhash: 5e68da4372238720dd46caae3b03787e
+    binhash: d1546a2b957a23e34f1154ad180df60d
     md5: b2bd35c44017597ba99b85fb61ed4d72
 work/input/topog.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/topog.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/topog.nc
```

Comment:
```suggestion
  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/topog.nc
```

Reply:
This file is actually used by both `mom` and `cice`

Reply:
Yes, the file is used in both, so I guess it will be needed for the CICE6-WW3 configuration, where we are not supposed to include the mom input directory.